analytics-integrations/google-analytics/src/main/java/com/segment/analytics/internal/integrations/GoogleAnalyticsIntegration.java

```diff
@@ -73,6 +76,16 @@
     }
     // tracker.setSampleRate(settings.getDouble("siteSpeedSampleRate", 1));
     sendUserId = settings.getBoolean("sendUserId", false);
+
+    customDimensions = settings.getValueMap("dimensions");
```

Comment:
This should already be an empty map coming from the server? No need to have the `null` check below.